kse/res/kse.sh

```diff
@@ -37,5 +37,5 @@ JAVA_OPTIONS="-splash:${SCRIPT_DIR}/splash.png"
 if [ -d "${SCRIPT_DIR}/jre" ]; then
     "${SCRIPT_DIR}/jre/bin/java" ${JAVA_OPTIONS} -jar "${JAR_FILE}" "$@"
```

Comment:
The double quotes should be around JAVA_OPTIONS here as well then or am I missing something?

Reply:
Yes, I overlooked this line. Thanks